java/src/jmri/jmrix/loconet/LnTurnout.java

```diff
@@ -289,13 +289,17 @@ private void setKnownStateFromOutputStateOddReport() {
         newCommandedState(CLOSED + THROWN);
         if (getFeedbackMode() == MONITORING || getFeedbackMode() == DIRECT) {
             newKnownState(CLOSED + THROWN);
+        } else if (getFeedbackMode() == EXACT) {
+            newKnownState(INCONSISTENT);
         }
     }
     
     private void setKnownStateFromOutputStateReallyOddReport() {
         newCommandedState(0);
         if (getFeedbackMode() == MONITORING || getFeedbackMode() == DIRECT) {
             newKnownState(0);
+        } else if (getFeedbackMode() == EXACT) {
+            newKnownState(INCONSISTENT);
```

Comment:
I think this should also be 0 like MONITORING and DIRECT. This case isn't quite as strong as the one above, mostly it's from consistency across modes.

Reply:
I've gone through the code for regular (i.e. standard) EXACT feedback messages. They set INCONSISTENT in this case, so I think your original change was correct.  Sorry for the confusion.